README.rst

```diff
@@ -206,7 +206,7 @@ Run Tests
 Use these commands to run code style checks and the test suite::
 
     black -l 100 --check .
-    DJANGO_DEV=1 pytest
+    DJANGO_DEV=1 python -m pytest
```

Comment:
If you are in an activate virtualenv, pytest should work. Did you experience an issue here?

Reply:
Yes. As mentioned [here](https://stackoverflow.com/questions/35045038/how-do-i-use-pytest-with-virtualenv), it fails on the first run. I've generated an [asciinema](https://asciinema.org/a/k7gQuPDlDxSU3iPCl8k3sVEi7) to reproduce the issue as well.